CoreAnalyticView/test/unit/src/au/gov/asd/tac/constellation/views/analyticview/state/AnalyticViewStateNGTest.java

```diff
@@ -0,0 +1,222 @@
+/*
+ * Copyright 2010-2023 Australian Signals Directorate
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.views.analyticview.state;
+
+import au.gov.asd.tac.constellation.graph.Graph;
+import au.gov.asd.tac.constellation.views.analyticview.AnalyticConfigurationPane;
+import au.gov.asd.tac.constellation.views.analyticview.questions.AnalyticQuestion;
+import au.gov.asd.tac.constellation.views.analyticview.questions.AnalyticQuestionDescription;
+import au.gov.asd.tac.constellation.views.analyticview.questions.BestConnectsNetworkQuestion;
+import au.gov.asd.tac.constellation.views.analyticview.results.AnalyticResult;
+import au.gov.asd.tac.constellation.views.analyticview.translators.GraphVisualisationTranslator;
+import au.gov.asd.tac.constellation.views.analyticview.utilities.AnalyticException;
+import au.gov.asd.tac.constellation.views.analyticview.utilities.AnalyticUtilities;
+import au.gov.asd.tac.constellation.views.analyticview.visualisation.GraphVisualisation;
+import au.gov.asd.tac.constellation.views.analyticview.visualisation.SizeVisualisation;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.TimeoutException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import static org.mockito.Mockito.mock;
+import org.testfx.api.FxToolkit;
+import static org.testng.Assert.*;
```

Comment:
same here (i'm guessing this might be the case for any unit test classes using assert library)